python/lsst/validate/base/metric.py

```diff
@@ -64,39 +157,16 @@ class Metric(JsonSerializationMixin):
     reference_page = None
     """Page number in the document that specifies this metric (`int`)."""
 
-    parameters = dict()
-    """`dict` of named :class:`Datum` values that must be known when measuring
-    a metric.
-
-    Parameters can also be accessed as attributes of the metric. Attribute
-    names are the same as key names in `parameters`.
-    """
-
-    def __init__(self, name, description, operator_str,
-                 specs=None, parameters=None,
+    def __init__(self, name, description, unit, tags=None,
                  reference_doc=None, reference_url=None, reference_page=None):
         self.name = name
         self.description = description
+        self.unit = unit
+        self.tags = tags
```

Comment:
Might want it do

```python
if tags is None:
    self.tags = []
else:
    self.tags = tags
```

but maybe I'm missing a use-case for tags being None rather than length-zero.